### DISCUSSION TITLE
Is there any way to save KP Parameters?

### DISCUSSION THREAD

**Discussion by adailsonfilho** [KillahPotatoes/KP-Liberation#859] [Q&A] [answered]
Hi,

Is there any way I could save my KP-liberation mission parameters? Those ones I set while in the lobby, you know?

Great job with this project, everyone!

**Comment by krieghof** [comment, marked as answer]
On the Wiki there is an entry, that explains the save/load of parameters.
https://github.com/KillahPotatoes/KP-Liberation/wiki/EN_ConfigFiles#uimission_paramshpp
Hope this helps you out.
Have fun playing :)

**Reply by adailsonfilho** [reply]
Thank you!
